Makefile

```diff
@@ -50,3 +50,8 @@ format:
 
 coverage:
 	$(ECHO) $(ROOT)/hack/coverage.sh --html "$(COVERAGE)" $(addprefix ./,$(DIRS))
+
+.PHONY: apiserver
+apiserver:
+	go install -v github.com/kubernetes-incubator/service-catalog/cmd/service-catalog
```

Comment:
@martinmaly `go install` won't install deps. this is the thing that essentially does `go build -o $GOPATH/bin/${BINARY_NAME}`